### ISSUE TITLE
The page for sys.objects says type_desc = 'TABLE_TYPE' but it's actually 'TYPE_TABLE'

### ISSUE THREAD

**Issue by Jehoel** [MicrosoftDocs/sql-docs#8060]
I'm having a _Mandela moment_ - I've been using `sys.objects` and `CREATE TYPE x AS TABLE ()` in SQL for _years_ and only just now I noticed that the `sys.objects.type_desc` for table-types is actually `'TYPE_TABLE'` and **not** `'TABLE_TYPE'` - it seems the author of the article on `sys.objects` is under the same shared delusion:

https://docs.microsoft.com/en-us/sql/relational-databases/system-catalog-views/sys-objects-transact-sql?view=sql-server-ver16

Screenshot of the mistake:

![image](https://user-images.githubusercontent.com/1693078/189478481-dc1e85d8-410b-442f-8248-ee3690b5907a.png)

Screenshot of the real value in a real query in an Azure SQL database:

![image](https://user-images.githubusercontent.com/1693078/189478629-64252575-21e7-40cd-9280-10267d4a6042.png)

**Comment by rwestMSFT** [comment]
On it.

**Comment by Jehoel** [comment]
@rwestMSFT It's Saturday...

**Comment by Jehoel** [comment]
Oh, another issue is that a lot of the values have incorrect compatibility information.

For example...
* `SEQUENCE_OBJECT` was introduced in SQL Server 2012, but it's listed as part of the _all versions_ group.
* `EDGE CONSTRAINT` was introduced in SQL Server 2017.

**Comment by rwestMSFT** [comment]
@Jehoel So why are you on GitHub? 😄

I've made a note for next week to fix this.

**Comment by Jehoel** [comment]
@liljack17 Why did you close this issue?

**Comment by WilliamAntonRohm** [comment]
@Jehoel -- Dai, thank you for your follow-up.  The next publishing cycle is this afternoon, so please look for this update to appear live by tomorrow.

**Comment by rwestMSFT** [comment]
That's my mistake, @Jehoel. I had a comment ready to post and never clicked "Comment". It would have read as follows:

"Thanks again for your contribution! You should see the updated article in the next day or so. This issue will close once the change is merged."
